tutorials/hyperformula-integration.html

```diff
@@ -0,0 +1,259 @@
+<div class="static-content">
+
+  <div class="example-container clearfix">
+    <div class="notification info">We have created a <a href="/docs/next/tutorial-migration-guide.html">migration guide</a> for developers upgrading to v9. If you use the formula plugin please make sure you carefully read it before upgrading.</div>
+
+    <h3 id="overview">Overview</h3>
+    <p>
+      The <i>Formulas</i> plugin provides you an extensive calculation capabilities based on formulas with the spreadsheet notation. Under the hood it uses an engine called <a href="https://handsontable.github.io/hyperformula/">HyperFormula</a> created by the Handsontable team as an independent library to help developers build complex data management apps.
+    </p>
+    <p>
+      This plugin comes with a library of 386 functions grouped into categories, such as Math and trigonometry, Engineering, Statistical, Financial, and Logical. Thanks to them you can create complex data entry rules in business apps, and many more. Below are some ideas on what you can do with it:
+    </p>
+    <p>
+      <ul>
+        <li>Fully-featured spreadsheet apps</li>
+        <li>Smart documents</li>
+        <li>Educational apps</li>
+        <li>Business logic builders</li>
+        <li>Forms and form builders</li>
+        <li>Online calculators</li>
+        <li>Low connectivity apps</li>
+      </ul>
+    </p>
+
+    <h3 id="features">Features</h3>
+    <p>
+      <ul>
+        <li>High-speed formula calculations</li>
+        <li>Function syntax compatible with Excel and Google Sheets</li>
+        <li>A library of built-in functions available in 16 languages</li>
+        <li>Support for wildcard characters</li>
+        <li>Support for CRUD operations</li>
+        <li>Support for cross-sheet references</li>
+        <li>Support for multiple Handsontable instances</li>
+        <li>Uses GPU acceleration for better performance</li>
+      </ul>
+    </p>
+
+    <h3 id="available-options-and-methods">Available options and methods</h3>
+    <p>
+      For the list of available settings and methods, visit the <a href="/docs/next/Formulas.html">API reference</a>.
```

Comment:
Next?
```suggestion
      For the list of available settings and methods, visit the <a href="/docs/???/Formulas.html">API reference</a>.
```

Do we even have this page? It's marked as deprecated and the list or API are not updated

Reply:
Each `/next/` version should be replaced with `/<?js= version ?>/`.

Reply:
@budnix Even the one in `formula-support.html`? Wouldn't that make users looking at doc versions <9.0.0 404?

Reply:
In that case, the `formula-support.html` page should link to the latest version of the documentation where the old Formulas was supported, I guess.

Reply:
I why do we link to the old documentation for "available options and methods" ? New plugins should have it's own reference

Reply:
Actually, I was only referring the new notification popup https://github.com/handsontable/docs/pull/312/files#diff-0d66ee487cccbbeb169b541cc2dc653567bfe5c99820230121dceaf6e6d3c4b2R2 - that page viewed on, say, 8.0.0 would always lead to a 404 if the link there was `/docs/<?=js version ?>/demo-hyperformula-integration.html` (such as https://handsontable.com/docs/8.0.0/demo-gantt-chart.html is a 404 but https://handsontable.com/docs/1.14.0/demo-gantt-chart.html isn't).

I'll just change it to a constant 9.0.0 as the first version that does include `demo-hyperformula-integration.html`, I hope this will be fine.

https://github.com/handsontable/docs/pull/312/commits/38ffc3f1d8ea5e4836f72df3128384f36f25827d